src/Orc.Controls/Controls/StepBar/Views/StepBar.xaml

```diff
@@ -49,9 +49,7 @@
 
                 <ItemsControl.ItemTemplate>
                     <DataTemplate>
-                        <orccontrols:StepBarItem Number="{Binding Number}"
-                                                 Title="{Binding Title}"
-                                                 State="{Binding State}"
+                        <orccontrols:StepBarItem Item="{Binding}"
```

Comment:
Not sure why this is needed. Isn't the datacontext already IStepBarItem? Then why explicitly introducing a new Item property?